test/test_new_project.sh

```diff
@@ -2,7 +2,8 @@
 set -ex
 
 commands=(
-    "node index.js tmp-project"
+    "node index.js tmp-project --rust=false"
+    "node index.js tmp-project --vanilla --rust=false"
     "node index.js tmp-project --vanilla"
```

Comment:
need to pass `--rust` here given default flag has been flipped

Reply:
done